BackendApartmentReservation/BackendApartmentReservation/Checklists/LivingPlaceManager.cs

```diff
@@ -16,6 +18,17 @@ public LivingPlaceManager(IApartmentRepository apartmentRepository)
             _apartmentRepository = apartmentRepository;
         }
 
+        public async Task<ApartmentCreatedResponse> CreateApartment(CreateApartmentRequest apartmentRequest)
+        {
+            var apartment =
+                await _apartmentRepository.CreateApartment(apartmentRequest.Address, apartmentRequest.Rooms);
+
+            return new ApartmentCreatedResponse
+            {
+                ApartmentId = apartment.Id.ToString()
```

Comment:
Since FE doesnt care about apartments, there is no need to return ID.

Other notice. If you are returning ID, make sure its a GUID not int